code/modules/surgery/bodyparts/head.dm

```diff
@@ -33,6 +33,17 @@
 	var/lip_style = null
 	var/lip_color = "white"
 
+
+/obj/item/bodypart/head/examine(mob/user)
+	..()
+	if(brain?.suicided || brainmob?.suiciding)
+		to_chat(user, "There's a pretty dumb expression on [src]'s face; they must have really hated life. There is no hope of recovery.")
+	else if(brain?.brain_death || brainmob?.health <= HEALTH_THRESHOLD_DEAD)
```

Comment:
Will this condition trigger if `brainmob` is null (debrained head)?

Reply:
Nope